lbrynet/core/Session.py

```diff
@@ -124,7 +124,7 @@ def __init__(self, blob_data_payment_rate, db_dir=None,
 
         self.rate_limiter = rate_limiter
 
-        self.external_ip = '127.0.0.1'
+        self.external_ip = None
 
```

Comment:
You said this happens when UPNP is not enabled? 
Maybe log a warning here? 
Also does this still work if someone wants to test the DHT on their local network? 
( I should probablly run lbry-in-a-box testing for this)